src/reducers/__tests__/deleteEmptyThought.ts

```diff
@@ -66,23 +67,23 @@ it('do not delete thought with children', () => {
 
 })
 
-it('archive thought with hidden children', () => {
+it(`archive thought with hidden children - arvhive all children in cursor's parent`, () => {
 
   const steps = [
     newThought(''),
-    newSubthought('=one'),
-    cursorBack,
-    deleteEmptyThought,
+    newSubthought('1'),
+    archiveThought({}),
+    deleteEmptyThought
   ]
```

Comment:
I think this might be clearer if we did it with a single `importText` and `setCursor` before calling `deleteEmptyThought`. I see how `archiveThought` sets up the condition of having a hidden child, but it took me a minute to figure out that it was being used to set up the test case.

Reply:
Also, let's use an inert hidden thought like `=one` or `=test` rather than `=archive`. `=archive` is a special case and usually is deleted permanently when it is archived.